src/bun.js/bindings/v8/V8HandleScopeBuffer.cpp

```diff
@@ -30,7 +30,7 @@ void HandleScopeBuffer::visitChildrenImpl(JSCell* cell, Visitor& visitor)
     for (int i = 0; i < thisObject->size; i++) {
         auto& handle = thisObject->storage[i];
         if (handle.isCell()) {
```

Comment:
are these zero initialized?  or whats the default vlaue?

Reply:
It's not super clear but `to_v8_object` and `map` get initialized to null pointers, while `ptr` is uninitialized. I'll improve the default `Handle` constructor on the same branch where I make the other changes to `HandleScopeBuffer` we were talking about.